common4j/src/main/com/microsoft/identity/common/java/crypto/AsymmetricAlgorithm.java

```diff
@@ -20,36 +20,28 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-package com.microsoft.identity.common.java.eststelemetry;
-
-import com.microsoft.identity.common.java.interfaces.INameValueStorage;
-
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
+package com.microsoft.identity.common.java.crypto;
 
 import lombok.NonNull;
 
-class InMemoryTelemetryMap implements INameValueStorage<CurrentRequestTelemetry> {
-
-    final ConcurrentMap<String, CurrentRequestTelemetry> mMap = new ConcurrentHashMap<>();
-
-    @Override
-    public CurrentRequestTelemetry get(final String name) {
-        return mMap.get(name);
-    }
-
-    @Override
-    public void put(final String name, final CurrentRequestTelemetry value) {
-        mMap.put(name, value);
-    }
-
-    @Override
-    public void remove(@NonNull String name) {
-        mMap.remove(name);
-    }
-
+/**
+ * Marker interface for asymmetric algorithms.
```

Comment:
What does Marker mean here?

Reply:
I don't know lol :D

Reply:
A marker interface generally has no methods.  I see this one evolved since I created it, and the javadoc should probably be corrected.  Also, there's no need to override interface methods when extending...